### PR TITLE
 [QPE-223] Setting up build pipeline

### PR DESCRIPTION
**Pull request by marols**
This PR contains:

QPE-226 Package files, create release and upload zip
QPE-228 Rename extension to "qlik-word-cloud"
QPE-229 Add/update license file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "qlik-word-cloud",
+  "version": "0.0.1",
+  "description": "Qlik Sense Extension Word Cloud",
+  "keywords": "word cloud qliksense extension",
+  "repository": "https://github.com/cleveranjos/br.com.clever.wordcloud",
```
Comment (lebond) [inline package.json]: Sound this be the Qlik fork?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "author": "Clever Anjos <clever@clever.com.br>",
```
Comment (lebond) [inline package.json:7]: replace with Qlik or remove?

**Review by lebond — APPROVED** [review]

File: scripts/bump-version.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# bump minor
+ARRAY_VERSION[1]=$((ARRAY_VERSION[1]+1))
+
+# join into string
+NEW_VERSION=$(join_by . ${ARRAY_VERSION[@]})
+echo "$NEW_VERSION"
+
+
+# Usage
+# $ bump-version.sh qlik-oss qsSimpleKPI
```
Comment (lebond) [inline scripts/bump-version.sh:23]: not important but so long as I noticed it..

File: scripts/create-release.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+set -o errexit
+
+echo "Bumping version from: $(scripts/print-version.sh $1 $2)"
+VERSION="$(scripts/bump-version.sh $1 $2)"
+echo "to: $VERSION"
+"${HOME}/bin/ghr" -t "${ghoauth}" -u "${CIRCLE_PROJECT_USERNAME}" -r "${CIRCLE_PROJECT_REPONAME}" -c "${CIRCLE_SHA1}" -delete "${VERSION}" "./build/$3.zip"
+
+
+# Usage
+# $ create-release.sh qlik-oss qsSimpleKPI qlik-multi-kpi
```
Comment (lebond) [inline scripts/create-release.sh:11]: also a candidate to be removed or updated

File: scripts/get-latest-version.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+if [ -z "${VERSION}" ]; then
+  VERSION="0.1.0"
+fi
+
+echo "${VERSION}"
+
+### Inspired by https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+# Usage
+# $ get-latest-version.sh qlik-oss qsSimpleKPI
```
Comment (lebond) [inline scripts/get-latest-version.sh:16]: here too

File: src/qlik-word-cloud.qext [hunk 6]
```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Word cloud chart",
+	"description": "Word Cloud",
+	"type": "visualization",
+	"icon": "treemap",
+	"version": "X.Y.Z",
+	"preview": "wordcloud.png",
+	"keywords": "qlik-sense, visualization",
+  "author": "Clever Anjos (clever@clever.com.br)",
```
Comment (lebond) [inline src/qlik-word-cloud.qext:9]: also remove?

**Review by reruined — APPROVED** [review]
I fixed the blocking bugs and updated the demo file to include both the old and new versions of the extension side-by-side. Works!

**Pull request merged**